src/topology/opens.lean

```diff
@@ -109,6 +113,22 @@ end
 lemma supr_def {ι} (s : ι → opens α) : (⨆ i, s i) = ⟨⋃ i, s i, is_open_Union $ λ i, (s i).2⟩ :=
 by { ext, simp only [supr, opens.Sup_s, sUnion_image, bUnion_range], refl }
 
+@[simp] lemma supr_s {ι} (s : ι → opens α) : ((⨆ i, s i : opens α) : set α) = ⋃ i, s i :=
+by simp [supr_def]
+
+@[simp] theorem mem_Union {ι} {x : α} {s : ι → opens α} : x ∈ supr s ↔ ∃ i, x ∈ s i :=
```

Comment:
This is a fine lemma, but I don't think it should be `simp`. It breaks through a (mild) "barrier" I think. So I would just dump the attribute.
```suggestion
theorem mem_Union {ι} {x : α} {s : ι → opens α} : x ∈ supr s ↔ ∃ i, x ∈ s i :=
```